packages_rs/nextclade-cli/src/cli/nextclade_cli.rs

```diff
@@ -835,7 +835,6 @@ pub fn nextclade_get_output_filenames(run_args: &mut NextcladeRunArgs) -> Result
     if output_selection.contains(&NextcladeOutputSelection::Translations) {
       let output_translations_path =
         default_output_file_path.with_file_name(format!("{output_basename}.cds_translation.{{cds}}.fasta"));
-      let output_translations_path = add_extension(output_translations_path, "translation.fasta");
```

Comment:
I think it's better to keep `add_extension()` call, just change both the strings.

I no longer remember details anymore, but there were weirdities when filename contains dots: https://github.com/nextstrain/nextclade/issues/907

This is the original (erroneous) commit: https://github.com/nextstrain/nextclade/commit/aea4cde536c5c7624c3d0170fe4b5a479b5bbd2f

Reply:
I'll see whether I can figure it out, thanks for the pointer to prevent a regression!

Reply:
This PR currently doesn't have the issue reported in #907 (i.e. it doesn't regress):
```
❯ cargo run --bin nextclade -- run -d sars-cov-2 --output-all out --output-basename basename.THISWILLDISAPPEAR -t out/test.tsv -- data/sars-cov-2/sequences.fasta
    Finished dev [unoptimized + debuginfo] target(s) in 0.18s
     Running `target/debug/nextclade run -d sars-cov-2 --output-all out --output-basename basename.THISWILLDISAPPEAR -t out/test.tsv -- data/sars-cov-2/sequences.fasta`

nextclade_master on  fix/output-translation-path [$!?] via  v20.5.1 via 🐍 v3.11.7 via 🦀 v1.70.0 via 🅒 py11 took 5s 
❯ ls out
basename.THISWILLDISAPPEAR.aligned.fasta                basename.THISWILLDISAPPEAR.cds_translation.ORF7b.fasta
basename.THISWILLDISAPPEAR.auspice.json                 basename.THISWILLDISAPPEAR.cds_translation.ORF8.fasta
basename.THISWILLDISAPPEAR.cds_translation.E.fasta      basename.THISWILLDISAPPEAR.cds_translation.ORF9b.fasta
basename.THISWILLDISAPPEAR.cds_translation.M.fasta      basename.THISWILLDISAPPEAR.cds_translation.S.fasta
basename.THISWILLDISAPPEAR.cds_translation.N.fasta      basename.THISWILLDISAPPEAR.csv
basename.THISWILLDISAPPEAR.cds_translation.ORF1a.fasta  basename.THISWILLDISAPPEAR.json
basename.THISWILLDISAPPEAR.cds_translation.ORF1b.fasta  basename.THISWILLDISAPPEAR.ndjson
basename.THISWILLDISAPPEAR.cds_translation.ORF3a.fasta  basename.THISWILLDISAPPEAR.nwk
basename.THISWILLDISAPPEAR.cds_translation.ORF6.fasta   test.tsv
basename.THISWILLDISAPPEAR.cds_translation.ORF7a.fasta
```

Reply:
Done in 56b04356bb351a355a73e47137b7a8b47dc9d859

Didn't observe any change but better be safe :)

Now good?